go.mod

```
@@ -1,3 +1,17 @@
+// NB: Since this repostiory was scaffolded using `ignite s chain pocket`
+// but the module was later rename to github.com/pokt-network/poktroll, we
```

Question: Is this the most recent state of affairs, incl. the feedback from rollkit team, etc.?

NIT: typo

```suggestion
// but the module was later renamed to github.com/pokt-network/poktroll, we
```